client/state/themes/reducer.js

```diff
@@ -227,6 +225,10 @@ export const queries = ( () => {
 		[ THEMES_RECEIVE ]: ( state, { siteId, themes } ) => {
 			return applyToManager( state, siteId, 'receive', true, themes );
 		},
+		[ THEME_REQUEST_FAILURE ]: ( state, { siteId, themeId, error } ) => {
+			const theme = [ { id: themeId, error } ];
+			return applyToManager( state, siteId, 'receive', true, theme );
+		},
```

Comment:
I don't think we should (ab)use the query manager to store a fake `theme` object containing an error. We could consider adding a dedicated `themeRequestErrors` reducer -- but do we even need to store the error?